app/views/merchant/dashboard/index.html.erb

```diff
@@ -2,8 +2,14 @@
 <p><%= @merchant.address %></p>
 <p><%= @merchant.city %> <%= @merchant.state %> <%= @merchant.zip %></p>
 
+<%= link_to 'Create Bulk Discount', new_merchant_discount_path %><br>
+
 <%= link_to 'My Items', '/merchant/items' %>
 
+<% @merchant.discounts.each do |discount| %>
```

Comment:
Nice job of displaying, but make sure to add functionality to the cart and actual order_items